### PR TITLE
Add Waveshare 7 and 4.3B and 5 and 5B

### PR DESCRIPTION
**Pull request by H-sw123**
Add support for [Waveshare ESP32-S3-Touch-LCD-7](https://www.waveshare.com/wiki/ESP32-S3-Touch-LCD-7) board
Add support for [Waveshare ESP32-S3-Touch-LCD-4.3B](https://www.waveshare.com/wiki/ESP32-S3-Touch-LCD-4.3B) board
Add support for [Waveshare ESP32-S3-Touch-LCD-5](https://www.waveshare.com/wiki/ESP32-S3-Touch-LCD-5) board
Add support for [Waveshare ESP32-S3-Touch-LCD-5B](https://www.waveshare.com/wiki/ESP32-S3-Touch-LCD-5) board

### CODE REVIEW

File: src/board/waveshare/ESP32_S3_Touch_LCD_4_3.h
Comment (Lzw655) [inline src/board/waveshare/ESP32_S3_Touch_LCD_4_3.h:1]: Hi @martinroger, do you think the modifications of this file are ok?

**Review by Lzw655 — COMMENTED** [review]
Hi @H-sw123, thank you for your contributions! I will create a new PR that picks your commits and merge them ASAP.

File: src/board/waveshare/ESP32_S3_Touch_LCD_4_3.h
Comment (Lzw655) [reply src/board/waveshare/ESP32_S3_Touch_LCD_4_3.h:1]: Alright, no worries. Thank you for checking!

**Comment by B4stl3r** [comment]
@H-sw123 
as you created the PR, maybe you could support me on:
https://github.com/esp-arduino-libs/ESP32_Display_Panel/issues/145

**Pull request closed without merging**